File: 1ºAno/Lista2/Python/Sala/files/ex4.py
a = float(input("Entre com valores referentes aos lados de um triângulo, eu direi:\n\na) Se os lados formam, de fato, um triângulo\n\nEm caso afirmativo, direi se o triângulo é:\n\nb) Equilátero\nc) Isósceles\nd) Escaleno\n\nL1 = "))
b = float(input("L2 = "))
c = float(input("L3 = "))
if a < b + c and b < a + c and c < a + b:
    if a == b and b == c:
        print ("O triângulo é equilátero :)")
    elif a == b or a == c or b == c:
        print ("O triângulo é isósceles :)")
    else:
        print ("O triângulo é escaleno ;)")
else:
    print ("Os valores fornecidos não formam um triângulo :/")

File: 1ºAno/Lista2/C/Sala/files/ex4.c
#include <stdio.h>

int main() {

    double a, b, c;
    printf ("Entre com os lados do triangulo, vossa senhoria:\n");
    printf ("A = ");
    scanf ("%lf", &a);
    printf ("B = ");
    scanf ("%lf", &b);
    printf ("C = ");
    scanf ("%lf", &c);
    if (a < b + c && b < a + c && c < b + a) {
        if (a == b && b == c) {
            printf ("Os valores dados formam um triangulo equilátero\n");
        }
        else if (a == b || a == c || b == c) {
            printf ("Os valores dados formam um triangulo isósceles\n");
        }
        else {
            printf ("Os valores dados formam um triangulo escaleno\n");
        }
    }
    else {
        printf ("Os valores dados não formarm um triangulo :(\n");
    }

    return 0;
}